Dockerfile

```diff
@@ -1,13 +1,14 @@
 FROM python as nasz_python
 
-ADD ./requirements.txt .
+COPY . .
```

Comment:
@thinkofher odradzam, lepiej wrzucić kod w katalog src albo coś. Tak jak to zrobiłeś teraz, to nawet byle zmiana w README.md inwaliduje Ci cały kontener.

Reply:
(w sensie, doczytaj o cache - Docker cachuje FS dla wszystkich każdej komendy osobno jako "warstwy" i jeśli np zrobisz najpierw `ADD . .` a potem `apt-get install cośtam`, to zmiana jakiegokolwiek pliku spowoduje że cache nie zostanie użyty dla apt-geta - dlatego requirements.txt potraktowałem specjalnie - najpierw je zainstaluj, potem dodaj resztę kodu, ale pomiń wszystkie nie-kodowe pliki jak Dockerfile albo README)

Reply:
w sumie jak tak myślę to może warto wrzucić to jako komentarz bo to nie jest oczywiste najwyraźniej ;)